app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+
+<div class="flex flex-col items-center mb-10">
+<%= render "header" %>
+  <% @groups.each_with_index do |group, index| %>
+    <% if current_user && group.user_id == current_user.id %>
+      <%= link_to group_entities_path(group.id), class: do %>
+      <div class="flex bg-slate-50 group-card rounded p-4 mt-4 w-94">
+        <%= image_tag group.icon, alt: 'icon', class: "w-28" %>
+
+        <div class="ml-5 grow text-gray-500 font-medium">
+          <p class="text-lg"><%= group.name %></p>
+          <p class="mt-14 text-xs"><%= group.created_at.strftime('%d %h %Y') %></p>
+        </div>
+
+        <p class="ml-16 grow-0 font-semibold">$<%= @total_amount[index] %> </p>
+      </div>
+      <% end %>
+    <% end %>
+  <% end %>
+
+  <%= link_to 'Add new category', new_group_path, class: "bg-green-400 p-2 w-96 text-center rounded text-slate-50 mt-6" %>
```

Comment:
- [ ] Kindly ensure the new category button sticks to the bottom of the page as described in the design.

- [ ] Kindly remove the sign-out button from the bottom of all your pages. Sign-out buttons do not belong there. They should be on the navbar.

  ![siss](https://github.com/KUBAHO3/budget-app/assets/54848363/3452b3a0-35c7-47cc-bad7-cbf5aabe34a2)